src/node/miner.cpp

```
@@ -19,9 +19,9 @@
 #include <policy/policy.h>
 #include <pow.h>
 #include <primitives/transaction.h>
-#include <timedata.h>
 #include <util/moneystr.h>
 #include <util/system.h>
+#include <util/time.h>
```

Cppcheck (2.19.0)

[error] `#error` No known always_inline attribute for this platform.

(preprocessorErrorDirective)